package.json

```diff
@@ -2,7 +2,9 @@
   "name": "webpack-dev-server",
   "version": "3.11.0",
   "description": "Serves a webpack app. Updates the browser on changes.",
-  "bin": "bin/webpack-dev-server.js",
+  "bin": {
+    "webpack-dev-server": "bin/webpack-dev-server.js"
```

Comment:
Just realized that `npm install` automatically changed this line for me. Running `webpack-dev-server` as a binary still functions the same (no breaking changes - https://docs.npmjs.com/cli/v6/configuring-npm/package-json#bin).

I can revert this line if desired.